stable/oauth2-proxy/values.yaml

```diff
@@ -1,4 +1,10 @@
+htpasswd:
+  enabled: false
+  # Content of the htpasswd file, as described here :
+  # https://github.com/pusher/oauth2_proxy/blob/master/contrib/oauth2_proxy.cfg.example#L58
+  htpasswd: "Eiph3Uvu:{SHA}DZsA/VT1TvJjNQC5ZcIMLwCiPx4="
 # Oauth client configuration specifics
+isOpenshift: false
```

Comment:
Is this a pattern existing in other Helm Charts?

Reply:
Hello Maxime, 
No, this does not.
I started to use it first in the official cert-manager chart here :
https://github.com/jetstack/cert-manager/tree/master/deploy/charts/cert-manager